server/aleph.ts

```diff
@@ -105,7 +105,8 @@ export class Aleph implements IAleph {
       Object.assign(this.#importMap, getDefaultImportMap())
     }
     if (configFile) {
-      Object.assign(this.#config, await loadConfig(configFile))
+      const config = await loadConfig(configFile)
+      Object.assign(this.#config, config, config.i18n)
```

Comment:
not needs to assign `config.i18n`

Reply:
When the `ssr` option in aleph.config.ts is `true`, the result of SSR is "Page Not Found". The final rendering result is fine, because it is re-rendered on the client side.

The result of SSR before the change.

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charSet="utf-8" />
  <meta name="viewport" content="width=device-width" />
  <link rel="modulepreload" href="/_aleph/-/http_localhost_2020/framework/core/module.js" />
  <link rel="modulepreload" href="/_aleph/-/http_localhost_2020/framework/core/events.js" />
  <link rel="modulepreload" href="/_aleph/-/http_localhost_2020/framework/core/routing.js" />
  <link rel="modulepreload" href="/_aleph/-/http_localhost_2020/framework/core/hmr.js" />
  <link rel="modulepreload" href="/_aleph/-/http_localhost_2020/framework/react/bootstrap.js" />
  <link rel="modulepreload" href="/_aleph/-/http_localhost_2020/shared/util.js" />
  <link rel="modulepreload" href="/_aleph/app.js" />
  <link rel="modulepreload" href="/_aleph/main.js" />
</head>
<body>
  <!-- Rendered as "Page Not Found". -->
  <div id="__aleph"><div style="padding:0;margin:0;line-height:1.5;font-size:15px;font-weight:400;color:#333;position:fixed;display:flex;align-items:center;justify-content:center;flex-direction:column;width:100vw;height:100vh"><p style="padding:0;margin:0;line-height:1.5;font-size:15px;font-weight:500;color:#333"><code style="padding:0;margin:0;line-height:1.5;font-size:15px;font-weight:700;color:#333">404</code><small style="padding:0;margin:0;line-height:1.5;font-size:14px;font-weight:400;color:#999"> - </small><span>Page Not Found</span></p></div></div>
  <script></script>
  <script src="/_aleph/main.js" type="module"></script>
  <script src="/_aleph/-/http_localhost_2020/nomodule.js" nomodule></script>
</body>
</html>
```

After the change.

```diff
- Object.assign(this.#config, await loadConfig(configFile))
+ const config = await loadConfig(configFile)
+ Object.assign(this.#config, config, config.i18n)
```

```html
<!DOCTYPE html>
<html lang="en">
<head>
  <meta charSet="utf-8" />
  <meta name="viewport" content="width=device-width" ssr />
  <title ssr>Hello World - Aleph.js</title>
  <style type="text/css" data-module-id="/style/index.css" ssr>...</style>
  <link rel="modulepreload" href="/_aleph/-/http_localhost_2020/framework/core/module.js" />
  <link rel="modulepreload" href="/_aleph/-/http_localhost_2020/framework/core/events.js" />
  <link rel="modulepreload" href="/_aleph/-/http_localhost_2020/framework/core/routing.js" />
  <link rel="modulepreload" href="/_aleph/-/http_localhost_2020/framework/core/hmr.js" />
  <link rel="modulepreload" href="/_aleph/-/http_localhost_2020/framework/react/bootstrap.js" />
  <link rel="modulepreload" href="/_aleph/-/http_localhost_2020/shared/util.js" />
  <link rel="modulepreload" href="/_aleph/app.js" />
  <link rel="modulepreload" href="/_aleph/pages/index.js" />
  <link rel="modulepreload" href="/_aleph/main.js" />
</head>
<body>
  <div id="__aleph"><main><div class="page"><p class="logo"><img src="/logo.svg" height="75" title="Aleph.js"/></p><h1>Welcome to use <strong>Aleph.js</strong>!</h1><p class="links"><a href="https://alephjs.org" target="_blank">Website</a><span></span><a href="https://alephjs.org/docs/get-started" target="_blank">Get Started</a><span></span><a href="https://alephjs.org/docs" target="_blank">Docs</a><span></span><a href="https://github.com/alephjs/aleph.js" target="_blank">Github</a></p><div class="counter"><span>Counter:</span><em>...</em><button>-</button><button>+</button></div><p class="copyinfo">Built by Aleph.js in Deno <!-- -->1.14.0</p></div></main></div>
  <script id="ssr-data" type="application/json">
{
  "useDeno-6G3UgSqfHlVo85M60IRbxdFdQao": {
    "value": "1.14.0",
    "expires": 0
  }
}
  </script>
  <script src="/_aleph/main.js" type="module"></script>
  <script src="/_aleph/-/http_localhost_2020/nomodule.js" nomodule></script>
</body>
</html>
```

Sorry, I did not include this in my comment.

Reply:
oh, i see! i changed the `Config` types but did not update the `RouterOptions`, check https://github.com/alephjs/aleph.js/blob/5c76d08ca55e14a3e4eadd90c9c5dd3e2c4d57cb/framework/core/routing.ts#L14